keps/sig-node/5554-in-place-update-pod-resources-alongside-static-cpu-manager-policy/README.md

```diff
@@ -0,0 +1,1210 @@
+* Will any other components on the node change? For example, changes to CSI, 
+  CRI or CNI may require updating that component before the kubelet.
+-->
+
+Not applicable.
+
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable; can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
+
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml` . This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[ `prod-readiness-approvers` ](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+<!--
+Pick one of these and delete the rest.
+
+Documentation is available on [feature gate lifecycle] and expectations, as
+well as the [existing list] of feature gates.
+
+[feature gate lifecycle]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md
+[existing list]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
+-->
+
+* [x] Feature gate (also fill in values in `kep.yaml`)
+  + Feature gate name: `InPlacePodVerticalScalingExclusiveCPUS`
+  + Components depending on the feature gate: kubelet, kube-controller-manager, kube-scheduler
+    - kubelet
+  + Requires `InPlacePodVerticalScaling` and `CPUManager` feature to be enabled
+  + Requires `--cpu-manager-policy` kubelet configuration set to “static” 
+  + Introduced in v1.33, alpha
+* [x] Other
+  + Describe the mechanism: 
+ + Will enabling / disabling the feature require downtime of the control
+    plane?
+    - No. 
+  + Will enabling / disabling the feature require downtime or reprovisioning
+    of a node?
+    - No.
+
+###### Does enabling the feature change any default behavior?
+
+<!--
+Any change of default behavior may be surprising to users or break existing
+automations, so be extremely careful here.
+-->
+
+Yes. Admission flow changes for In-place CPU resize requests for Guaranteed QOS pods. Kubelet will start processing these requests and adjust the CPU Manager's internal state and potentially the CPU and/or NUMA affinity cgroup to apply the resize.
```

Comment:
I think the answer is actually no - static CPU manager and InPlacePodVerticalScaling are both opt-in features (as in, it requires user initiation to trigger the feature) so there is no "default" behavior that will change or be broken.